content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -32,19 +32,24 @@ mitigate some future vulnerabilities too.
 {{% thirdparty-content single="true" %}}
 <!-- if adding another runtime in the future, omit the single setting -->
 
-This is a Linux only feature. In addition, support is needed in the 
+This is a Linux only feature. In addition, support is **needed** in the 
 {{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}
 to use this feature with Kubernetes stateless pods:
 
-* CRI-O: v1.25 has support for user namespaces.
+* CRI-O: supports user namespaces with all Kubernetes versions >= v1.25
+* containerd: v1.7 supports user namespace with Kubernetes v1.25 and v1.26.
 
-* containerd: support is planned for the 1.7 release. See containerd
-  issue [#7063][containerd-userns-issue] for more details.
+Containerd plans to add support for user namespaces with Kubernetes >= 1.27 in
+version 1.8. For more information, see the [containerd issue][containerd-issue].
 
 Support for this in [cri-dockerd is not planned][CRI-dockerd-issue] yet.
 
+You should not use user namespaces support in Kubernetes >= 1.27 with
+containerd 1.7. The Kubernetes implementation was reworked and has requirements
+that containerd 1.7 doesn't support.
```

Comment:
That change needs to land on the branch _dev-1.27_ and not here.

Reply:
ok, removed